services/frontend/helm/values.yaml

```diff
@@ -1,6 +1,6 @@
 
 image:
-  repository: dev.local/frontend
+  repository: ghcr.io/yousofmersal/dm885_frontend
```

Comment:
will this cause issues when running locally?

Reply:
Jeg synes jeg læste på Skaffold docs at det er sådan det er ment man skal bruge det, men det er helt sikkert en god idé lige at teste det inden vi merger.

Den skal nok også lige opdateres i `skaffold.yaml` filerne.
https://github.com/YousofMersal/DM885_Project_DevSecOps/blob/cea3176a72112ac3ad3335732beffdcc521ec463/services/frontend/skaffold.yaml#L5-L9